CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.10.2
```

Comment:
Since #135 is ready too, you could simply set this as `master`. Because it won't be published until #135

```md
## master
### Changed
* My change....

## 0.3.10.1
.
.
.
```

Reply:
Done! Thanks for the suggestion :smile: